requirements/base.txt

```diff
@@ -15,3 +15,5 @@ mysqlclient==1.3.12
 pytz==2018.3
 wagtail>=1.13,<1.14
 django-cors-headers==2.3.0
+django-object-actions==0.10.0
```

Comment:
I think we can avoid using if we override `change_form` [template for journal model](https://docs.djangoproject.com/en/1.11/ref/contrib/admin/#overriding-vs-replacing-an-admin-template)